api/src/services/graphql/schema/read.ts

```diff
@@ -734,7 +734,7 @@ export function getReadableTypes(
 					args: {
 						event: subscriptionEventType,
 					},
-					subscribe: createSubscriptionGenerator(gql, eventName),
+					subscribe: await createSubscriptionGenerator(gql, eventName),
```

Comment:
I think this should be removed

```suggestion
					subscribe: createSubscriptionGenerator(gql, eventName),
```